PE/__init__.py

```diff
@@ -1019,7 +1022,10 @@ def parseExports(self):
                     fwdname = self.readAtRva(funcoff, 260, shortok=True).split(b'\x00', 1)[0]
                     self.forwarders.append((funclist[ordl], name.decode('utf-8'), fwdname))
                 else:
-                    self.exports.append((funclist[ordl], ordl, name.decode('utf-8')))
+                    try:
+                        self.exports.append((funclist[ordl], ordl, name.decode('utf-8')))
+                    except UnicodeDecodeError:
+                        logger.info(f'Invalid export name at position {ordl}: {name}')
```

Comment:
```suggestion
                        logger.warning('Invalid name for export ordinal %d: %s', ordl, name[:8].decode('ascii', errors='ignore'))
```

Reply:
@williballenthin 
> `'ascii'`

why ascii? 😕 

Do we prefer to use `decode` with `error='replace'` than the bytes? I find the bytes provide more information:

```
>>> name[:16]
b'\xc2\x06\xc0,\x96?^\x16\xfe\x9e\xc0\x089\r\xd5x'
>>> name[:16].decode('utf-8', errors='replace')
'�\x06�,�?^\x16���\x089\r�x'
```

Reply:
How about something simple `name[:16].hex()`? Keep it simple without erasing any bytes.

Reply:
@rakuy0 

> name[:16].hex()

I like that! @williballenthin are you also ok with that?